backfill_corrections/delphiBackfillCorrection/NAMESPACE

```diff
@@ -29,9 +31,11 @@ importFrom(dplyr,filter)
 importFrom(dplyr,group_by)
 importFrom(dplyr,group_split)
 importFrom(dplyr,if_else)
+importFrom(dplyr,mutate)
 importFrom(dplyr,pull)
 importFrom(dplyr,select)
 importFrom(dplyr,summarize)
+importFrom(dplyr,ungroup)
 importFrom(evalcast,weighted_interval_score)
```

Comment:
@jingjtang `evalcast` takes a long time to install because it has a lot of dependencies. Would it be possible to define our own `weighted_interval_score` instead of importing it? How important is making sure that our definition matches that in evalcast?

Reply:
I would say it's important. @brookslogan  Does the forecast team also use `weighted_interval_score` from `evalcast`?

Reply:
Okay, thanks for verifying 👍

Reply:
Yes, we use `weighted_interval_score` from `evalcast`.  Matching its mathematical definition is very important, as this is a metric the community has developed for some evaluations.  It would be nice to have a faster implementation.  It would also be nice to have an implementation outside of `evalcast` since we're going to replace some of its functionality with `epiprocess`, but I don't know if that place should be `epiprocess` or something else.